### ISSUE TITLE
Where is mcloader defined?

### ISSUE THREAD

**Issue by Betty-J** [Omid-Nejati/MedViT#11]
![截屏2024-06-04 17 55 13](https://github.com/Omid-Nejati/MedViT/assets/23076443/a19db79d-da21-4c0e-b3d8-ff5e4c6331ce)

**Comment by Omid-Nejati** [comment]
main.py  >>> line 125

https://github.com/Omid-Nejati/MedViT/blob/main/CustomDataset/main.py

**Comment by Betty-J** [comment]
> main.py >>> line 125
> 
> https://github.com/Omid-Nejati/MedViT/blob/main/CustomDataset/main.py

Certainly, I understand the definition of this parameter. What I mean is that you have used **_from mcloader import ClassificationDataset_** here, but I haven't found any definition related to this part in MedVit. Did you implement the mcloader yourself?
